docs/src/plugins/scroll-tracking/scroll-tracking.js

```
@@ -0,0 +1,292 @@
+    if (window._scrollTrackingObserver._mutationObserver) {
+      window._scrollTrackingObserver._mutationObserver.disconnect();
+    }
+    
+    // Clean up intersection observer
+    window._scrollTrackingObserver.disconnect();
+    window._scrollTrackingObserver = null;
+  }
+  
+  // Clear tracking flags from elements
+  document.querySelectorAll('[data-scroll-tracked]').forEach(el => {
+    delete el._scrollTrackingObserved;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)